apps/condo/domains/organization/schema/Organization.test.js

```diff
@@ -319,41 +319,11 @@ describe('organization TIN: various cases',  () => {
 
     test('admin: create Organization with valid 12 digits RU INN and RU country code ', async () => {
         const admin = await makeLoggedInAdminClient()
-        const [createdOrganization] = await createTestOrganization(admin, { meta: { inn: VALID_RU_TIN_12 }, country: RUSSIA_COUNTRY })
+        const [createdOrganization] = await createTestOrganization(admin, { tin: VALID_RU_TIN_12, country: RUSSIA_COUNTRY })
 
         const organizationData = await Organization.getAll(admin, { id: createdOrganization.id })
 
         expect(organizationData).toHaveLength(1)
         expect(organizationData[0].tin).toEqual(VALID_RU_TIN_12)
     })
-
-    test('admin: create Organization with invalid 10 digits RU INN and RU country code ', async () => {
```

Comment:
wow! do not delete the tests %) just refactor them

Reply:
These tests are testing tin resolvers functionality (resolvers are validating tin before return it and set it to null in case if it not a valid tin). So now there are no resolvers - so we don't need these tests anymore

Reply:
I reverted it back with updated test login